$NetBSD: patch-libsecret_Makefile.am,v 1.1 2018/05/20 17:30:04 leot Exp $

Avoid to use not-so-ubiquitous/non-standard dollar single quotes use ($'...').

--- libsecret/Makefile.am.orig	2017-12-05 13:16:03.000000000 +0000
+++ libsecret/Makefile.am
@@ -89,7 +89,8 @@ temp-dbus-generated.c: $(DBUS_XML_DEFINI
 	$(AM_V_GEN) gdbus-codegen --interface-prefix org.freedesktop.Secret. \
 		--generate-c-code temp-dbus-generated --c-namespace SecretGen $<
 secret-dbus-generated.c: temp-dbus-generated.c Makefile.am
-	$(AM_V_GEN) sed -e $$'1i\\\n#define GLIB_DISABLE_DEPRECATION_WARNINGS' \
+	$(AM_V_GEN) sed -e '1i\
+#define GLIB_DISABLE_DEPRECATION_WARNINGS' \
 		-e 's/secret_gen_/_secret_gen_/g' -e 's/type-/type/g' \
 		-e 's/temp-dbus-generated.h/secret-dbus-generated.h/g' \
 		temp-dbus-generated.c > secret-dbus-generated.c
